v3/src/components/case-tile-common/attribute-header.tsx

```diff
@@ -200,13 +218,15 @@ export const AttributeHeader = observer(function AttributeHeader({
                     />
                   : <>
                       <MenuButton
-                          className="codap-attribute-button" ref={menuButtonRef}
+                          className={clsx("codap-attribute-button", {"table": instanceId.includes("table")})}
+                          ref={menuButtonRef}
                           disabled={attributeId === kIndexColumnKey}
                           sx={customButtonStyle}
                           fontWeight="bold" onKeyDown={handleButtonKeyDown}
                           data-testid={`codap-attribute-button ${attrName}`}
                           aria-describedby={`sr-column-header-drag-instructions-${instanceId}`}>
-                        {`${attrName ?? ""}${showUnits ? units : ""}`}
+                        {instanceId.includes("table") ? renderAttributeLabel
+                                                      : `${attrName ?? ""}${showUnits ? attrUnits : ""}`.trim()}
```

Comment:
Rather than determining whether to support two-line rendering by parsing the `instanceId`, I'd recommend passing in a boolean prop named something like `allowTwoLines` which defaults to `false` but the case table can pass `true`.